runelite-client/src/main/java/net/runelite/client/ui/overlay/components/BackgroundComponent.java

```diff
@@ -47,45 +47,49 @@
 	private Color backgroundColor = ComponentConstants.STANDARD_BACKGROUND_COLOR;
 	private Rectangle rectangle = new Rectangle();
 	private boolean fill = true;
+	private boolean border = true;
```

Comment:
This doesn't look used?

Reply:
It's used here:
https://github.com/runelite/runelite/blob/8129b369ae4fb9d71b40842233d48a5c36c3712c/runelite-client/src/main/java/net/runelite/client/ui/overlay/components/InfoBoxComponent.java#L102